tests/functional/disaster-recovery/regional-dr/test_2az_failover_relocate_zone_failure_test.py

```
@@ -0,0 +1,680 @@
+        )
+        logger.debug(f"Configured arbiter_zone='{arbiter_zone}'")
+
+        worker_nodes = get_nodes()
+        all_zone_labels = {
+            node.data["metadata"]["labels"][constants.ZONE_LABEL]
+            for node in worker_nodes
+            if constants.ZONE_LABEL in node.data["metadata"]["labels"]
+        }
+        logger.debug(
+            f"Found {len(worker_nodes)} worker nodes across "
+            f"{len(all_zone_labels)} zone(s): {sorted(all_zone_labels)}"
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)